test/tasks/ods-build-gradle_test.go

```diff
@@ -23,7 +23,8 @@ func TestTaskODSBuildGradle(t *testing.T) {
 						"sonar-quality-gate": "true",
 					}
 				},
-				WantRunSuccess: true,
+				RequiredServices: []string{"Bitbucket", "Nexus", "SonarQube"},
```

Comment:
Maybe we could use constants for this?

E.g. `const testBitbucketService = "localhost:7990"`

That would remove the need for the map in the other file, and you avoid typos.

Reply:
Thanks for the feedback. I thought about this, but wasn't sure where to define such a variable. Also how would I be able to log which service is unreachable if I only pass the URL?

I first wanted to go for an Enum like this but I faced the above mentioned problem as well:
```go
const (
  Bitbucket Service = "7990"
  Nexus             = "8081"
  SonarQube         = "9000"
)  
```